addons/website/views/website_templates.xml

```diff
@@ -393,7 +393,7 @@
                         </div>
                         <!-- Social -->
                         <div class="col-lg-6 pb16">
-                            <div class="s_social_media text-left no_icon_color" data-snippet="s_social_media" data-name="Social Media" contenteditable="false">
+                            <div class="s_social_media text-left no_icon_color o_not_editable" data-snippet="s_social_media" data-name="Social Media" contenteditable="false">
```

Comment:
Not sure if this is related to this commit, but you reintroduced the error where the user can select a zone between 2 icons and then type something in between (the error you told me about when I refactored the original PR).
Maybe worth to retrieve all those "this should not be allowed" small reviews and put them on top of the PR so we can always easily check which corner cases behavior should be controlled.